keyboards/xelus/pachi/rgb/rules.mk

```
@@ -0,0 +1,32 @@
+# MCU name
+MCU = STM32L433
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+
+EEPROM_DRIVER = i2c
+
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = custom
+
+COMMON_VPATH += $(DRIVER_PATH)/issi
+SRC += drivers/issi/is31fl3741.c
+
+LTO_ENABLE = yes
+OPT = 2
```

```suggestion
OPT = 2
```

Does `O2` provide smaller/faster code than the default `Os`?

---

Yes, O2 is faster code.